src/modules/scraper/adapter/messaging/FeeBreakdownConsumer.ts

```diff
@@ -31,8 +31,16 @@ export class FeeBreakdownConsumer {
       return;
     }
 
-    if (!deposit.token || !deposit.price || !deposit.depositDate) {
-      throw new Error("Can not get fee breakdown without token, price or deposit date");
+    if (!deposit.token) {
+      throw new Error("Token not populated");
+    }
+
+    if (!deposit.price) {
+      throw new Error("Price not populated");
+    }
```

Comment:
Do we have access to the SDK? If so, can we use `isDefined()`

Reply:
No, we currently don't use the SDK the Scraper. If we need it at some point, we could use the helper yea